tools/cbmc/proofs/ARP/ARPGenerateRequestPacket/ARPGenerateRequestPacket_harness.c

```diff
@@ -0,0 +1,28 @@
+/* FreeRTOS includes. */
+#include "FreeRTOS.h"
+#include "queue.h"
+
+/* FreeRTOS+TCP includes. */
+#include "FreeRTOS_IP.h"
+#include "FreeRTOS_IP_Private.h"
+#include "FreeRTOS_ARP.h"
+
+/*
+ * The assumption made here is that the buffer pointed by pucEthernerBuffer is at least
+ * allocated to sizeof(ARPPacket_t) size.
+ * This is not checked inside vARPGenerateRequestPacket.
+ */
+void harness()
+{
+	ARPPacket_t arp_packet;
+	NetworkBufferDescriptor_t pxNetworkBuffer;
```

Comment:
If `NetworkBufferDescriptor_t` is not a pointer type, please call this `xNetworkBuffer`.